packages/sitecore-jss-cli/package.json

```diff
@@ -2,8 +2,11 @@
   "name": "@sitecore-jss/sitecore-jss-cli",
   "version": "19.0.0-canary.24",
   "description": "Sitecore JSS command-line",
+  "main": "dist/cjs/cli.js",
+  "module": "dist/esm/cli.js",
+  "sideEffects": false,
   "scripts": {
-    "build": "npm run clean && tsc",
+    "build": "npm run clean && tsc -p tsconfig.json && tsc -p tsconfig-esm.json && tsc -p tsconfig-bin.json",
```

Comment:
As opposed to the separate bin compile using `tsconfig-bin.json`, couldn't we just point our bin to the location under "dist/cjs"? e.g.
```
"bin": {
    "jss": "dist/cjs/bin/jss.js"
  },
```

Reply:
Should `src/bin` be excluded from the `dist/esm` folder and only in `dist/cjs`?

Reply:
Changes pushed.